langkit/toxicity.py

```diff
@@ -1,13 +1,35 @@
 from typing import Optional
 
-from whylogs.core.datatypes import String
-from whylogs.experimental.core.metrics.udf_metric import register_metric_udf
+from whylogs.core.stubs import pd
+from whylogs.experimental.core.udf_schema import register_dataset_udf
+from . import LangKitConfig
+
+lang_config = LangKitConfig()
 
 _toxicity_model_path = "martin-ha/toxic-comment-model"
 _toxicity_tokenizer = None
 _toxicity_pipeline = None
 
 
+def toxicity(text):
```

Comment:
Should I pull this out into `toxicity(text: str) -> float` with a UDF wrapper as well?